docker.local/config/sc.yaml

```diff
@@ -149,11 +149,13 @@ smart_contracts:
     # reward paid out every block
     block_reward:
       block_reward: 1
+      block_reward_change_period: 10000
+      block_reward_change_ratio: 0.1
       qualifying_stake: 1
       sharder_ratio: 10
       miner_ratio: 40
-      blobber_capacity_ratio: 10
-      blobber_usage_ratio: 40
+      blobber_ratio: 50
+      challenge_period: 30
```

Comment:
Add calculation parameters (gamma, a, b, i, k, mu). Check slack thread https://0chain.slack.com/archives/C02MT6RV6QP/p1644840052762349?thread_ts=1644236441.266629&cid=C02MT6RV6QP for labels.